swarm/storage/memstore.go

```diff
@@ -55,7 +54,7 @@ func NewMemStore(params *StoreParams, _ *LDBStore) (m *MemStore) {
 	}
 
 	requestEvicted := func(key interface{}, value interface{}) {
```

Comment:
@gbalint I am afraid this was being called a lot when I ran tests with bigger files. Even though no errors were triggered, we were working under the assumption that requests are not evicted from the cache. We should re-think this.